.github/autobuild/windows.ps1

```diff
@@ -19,7 +19,7 @@ $ChocoCacheDir = 'C:\ChocoCache'
 $Qt32Version = "5.15.2"
 $Qt64Version = "5.15.2"
 $AqtinstallVersion = "2.1.0"
-$JackVersion = "1.9.17"
+$JackVersion = "1.9.21.20220626"
```

Comment:
Where did this version number come from?  The PR says just 1.9.21 without explanation.  This looks like a patch or nightly or some non-standard release.  We should have a good reason for that.

Reply:
It's "vanilla jack 1.9.21 as packaged for choco as of 20220626". The initial jack 1.9.21 choco package release did not work as expected (see above), which is why the released an updated package with this version.

This information is in a [collapsed conversation thread](https://github.com/jamulussoftware/jamulus/pull/2665#discussion_r913418469) above (I've just uncollapsed it due to its relevance).

The end-user relevant information is still that JACK has been updated to 1.9.21. Maybe we could have a code comment. Personally, I think this PR is sufficient as there is nothing special to remember when doing the next bump or something.

Reply:
No, not important - just I missed the collapse thread.  Thanks.